src/app/privacy-page/privacy-page.component.html

```diff
@@ -0,0 +1,87 @@
+<div class="info-section ">
```

Comment:
Isn't this a full standalone page? I expected us to either use find-support or help-center as templates.

Reply:
The doc contained content for the privacy page, but that page did not exist, and was marked as [todo](https://github.dev/CodeForAfrica/harassment-manager/blob/main/src/app/footer/footer.component.html#L53-L54) so we have to create it.

Reply:
Yes @kelvinkipruto. I meant I expected us to follow either [`help-center`](https://github.com/CodeForAfrica/harassment-manager/tree/main/src/app/help-center) or [`find-support`](https://github.com/CodeForAfrica/harassment-manager/tree/main/src/app/find-support) structure when creating the new page. Looking at the code on this PR, are we following any of the existing page structure?

Reply:
Yes, now I get you. Let me update it.